packages/core/src/Resource/Resource.js

```diff
@@ -36,13 +36,13 @@ import {
  * @returns {HyralResource}
  */
 function Resource(id = null, type = null, data = null, relationships = null) {
-  const state = [{
+  let state = [{
```

Comment:
State is never reassigned. It could still be const.

Reply:
I saw it later on, fixed in another PR